packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -1035,7 +1035,7 @@ class ObjectGroup extends InspectorObjectGroupBase {
     switch (treeType) {
       case FlutterTreeType.widget:
         newSelection = await invokeServiceMethodReturningNodeInspectorRef(
-          inLocalProject
+          inLocalProjectOnly
```

Comment:
I actually like the 'restrictToLocalProject' name that you suggested. I think this makes it more clear that the response is going to be restricted to a specific set of widgets.

Reply:
SG! Switch to `restrictToLocalProject`